spyder_kernels/customize/spydercustomize.py

```diff
@@ -422,6 +426,32 @@ def interaction(self, frame, traceback):
             self._cmdloop()
             self.forget()
 
+    def stop_here(self, frame):
+        """Check if pdb should stop here."""
+        if not super(SpyderPdb, self).stop_here(frame):
+            return False
+        filename = frame.f_code.co_filename
+        if filename.startswith('<'):
+            # This is not a file
+            return True
+        if self.pdb_ignore_lib and self.path_is_lib(filename):
+            return False
+        return True
+
+    def path_is_lib(self, path):
```

Comment:
We have a method in the `UserModuleReloader` class that does exactly this. So please unify these two code in a single function an use it in both here and there.

Reply:
Done